ArduPlane/GCS_Mavlink.cpp

```diff
@@ -699,6 +699,8 @@ bool GCS_MAVLINK_Plane::try_send_message(enum ap_message id)
     case MSG_AOA_SSA:
         CHECK_PAYLOAD_SIZE(AOA_SSA);
         plane.send_aoa_ssa(chan);
+    case MSG_LANDING:
+        plane.landing.send_landing_message(chan);
```

Comment:
Regarding checking flight stage here... It's redundant to do it before emitting the call to get here, and here and in the AP_Landing interface... I'd strongly only rather do the check in one place, and I think there is merit in just letting that be within landing. But it seems kinda silly to have it duplicated twice...